gateway/assets/script/bootstrap.js

```diff
@@ -36,24 +36,23 @@ app.controller("home", ['$scope', '$log', '$http', '$location', '$timeout', '$md
         }
 
         $scope.invocation.request = "";
-        $scope.isFunctionReady = "Querying";
-        $scope.fetchFunctionsDelay = 3500;
-        $scope.queryFunctionDelay = 2500;
+        var fetchFunctionsDelay = 3500;
+        var queryFunctionDelay = 2500;
         
-        $scope.fetchFunctionsInterval = setInterval(function() {
+        var fetchFunctionsInterval = setInterval(function() {
```

Comment:
`$scope` isn't needed and `fetchFunctionsInterval` isn't used either. But it does add readability so I think it could stay.

Reply:
The intent is to capture these variables so they can be cancelled if needed i.e. in debug mode.